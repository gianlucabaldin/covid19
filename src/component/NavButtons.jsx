import React from 'react';
import Button from '@material-ui/core/Button';
import { Box } from '@material-ui/core';

const NavButtons = () => {
  return (
    <Box display="flex" justifyContent="center" m={1} p={1}>
      <Box m={1}>
        <Button variant="outlined" color="primary" disabled>
          Country List
        </Button>
      </Box>
      <Box m={1}>
        <Button variant="outlined" color="primary">
          Worldwide
        </Button>
      </Box>
      <Box m={1}>
        <Button variant="outlined" color="primary">
          Europe
        </Button>
      </Box>
      <Box m={1}>
        <Button variant="contained" color="primary">
          Italy
        </Button>
      </Box>
    </Box>
  );
};

export default NavButtons;
