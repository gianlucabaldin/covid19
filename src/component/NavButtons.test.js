import React from 'react';
// import { render } from '@testing-library/react';
import { shallow } from 'enzyme';
import NavButtons from './NavButtons';

describe('NavButtons', () => {
  const wrapper = shallow(<NavButtons />);

  it('renders properly', () => {
    expect(wrapper).toMatchSnapshot();
  });
  /*
  it('has a left-aligned `Covid 19` text', () => {
    expect(wrapper.find('.MuiTypography-h6').exists()).toEqual(true);
    expect(wrapper).toIncludeText('Covid 19');
  });

  it('has a center-aligned project description - with Enzyme', () => {
    const { getByText } = render(<Header />);
    expect(getByText('Latest updates & historical data')).toBeInTheDocument();
  });

  // https://github.com/FormidableLabs/enzyme-matchers/blob/master/packages/jest-enzyme/README.md
  it('has a center-aligned project description - with jest-enzyme', () => {
    expect(wrapper).toIncludeText('Latest updates & historical data');
  }); */
});
