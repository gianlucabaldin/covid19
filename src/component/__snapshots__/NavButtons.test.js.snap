// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NavButtons renders properly 1`] = `
<Styled(MuiBox)
  display="flex"
  justifyContent="center"
  m={1}
  p={1}
>
  <Styled(MuiBox)
    m={1}
  >
    <WithStyles(ForwardRef(Button))
      color="primary"
      disabled={true}
      variant="outlined"
    >
      Country List
    </WithStyles(ForwardRef(Button))>
  </Styled(MuiBox)>
  <Styled(MuiBox)
    m={1}
  >
    <WithStyles(ForwardRef(Button))
      color="primary"
      variant="outlined"
    >
      Worldwide
    </WithStyles(ForwardRef(Button))>
  </Styled(MuiBox)>
  <Styled(MuiBox)
    m={1}
  >
    <WithStyles(ForwardRef(Button))
      color="primary"
      variant="outlined"
    >
      Europe
    </WithStyles(ForwardRef(Button))>
  </Styled(MuiBox)>
  <Styled(MuiBox)
    m={1}
  >
    <WithStyles(ForwardRef(Button))
      color="primary"
      variant="contained"
    >
      Italy
    </WithStyles(ForwardRef(Button))>
  </Styled(MuiBox)>
</Styled(MuiBox)>
`;
